applications/minotari_console_wallet/src/automation/commands.rs

```diff
@@ -711,10 +711,15 @@ pub async fn command_runner(
                         println!("Burnt {} Minotari in tx_id: {}", args.amount, tx_id);
                         println!("The following can be used to claim the burnt funds:");
                         println!();
-                        println!("claim_public_key: {}", proof.reciprocal_claim_public_key);
-                        println!("commitment: {}", proof.commitment.to_public_key()?);
-                        println!("ownership_proof: {:?}", proof.ownership_proof);
-                        println!("ownership_proof: {:?}", proof.range_proof);
+                        if let Some(proof) = proof {
+                            println!("claim_public_key: {}", proof.reciprocal_claim_public_key);
+                            println!("commitment: {}", proof.commitment.to_public_key()?);
```

Comment:
1. `BurnMinotari(args) => {` executes `match burn_tari(` which in turn calls onto `pub async fn burn_tari(`. In the match it short-curcuits `sidechain_deployment_key: Option<PrivateKey>,` to `None`.
2. In turn, `pub async fn burn_tari(` executes `wallet_transaction_service.burn_tari(`, but it short-curcuits `claim_public_key: Option<CompressedPublicKey>,` to `None`.
3. In it's current form, burning XTM via the command line, those funds will be lost forever.
4. `pub struct BurnMinotariArgs {` must make provision for `claim_public_key` and `sidechain_deployment_key` and those should be wired in.

```suggestion
                        if let Some(proof) = proof {
                            println!("claim_public_key: {}", proof.reciprocal_claim_public_key);
                            println!("commitment: {}", proof.commitment.to_public_key()?);
```

Reply:
Going to remove this feature as it's not used (discussed with @SWvheerden)